docker-compose/tomcat-manager/docker-compose.yml

```diff
@@ -0,0 +1,241 @@
+#      - traefik.http.routers.postgres-router.middlewares=test-stripprefix
+#      - traefik.http.middlewares.test-stripprefix.stripprefix.prefixes=/postgres
+    restart: always
+
+  elasticsearch:
+    image: 'elasticsearch:7.13.3'
+    container_name: elasticsearch
+    environment:
+      - discovery.type=single-node
+      - bootstrap.memory_lock=true
+      - ES_JAVA_OPTS=-Xms${ELASTICSEARCH_JAVA_MEMORY} -Xmx${ELASTICSEARCH_JAVA_MEMORY}
+      - xpack.security.enabled=false
+#    ports:
+#      - '${ELASTICSEARCH_PORT}:${ELASTICSEARCH_PORT}'
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+      nofile:
+        soft: 65536
+        hard: 65536
+      nproc:
+        soft: 4096
+        hard: 4096
+    volumes:
+      - type: volume
+        source: elasticsearch_data
+        target: /usr/share/elasticsearch/data
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "10m"
+        max-file: "10"
+        mode: non-blocking
+#    labels:
+#      - traefik.http.routers.elasticsearch-router.tls=true
+#      - traefik.http.routers.elasticsearch-router.entrypoints=https
+#      - traefik.http.routers.elasticsearch-router.rule=PathPrefix(`/elasticsearch`)
+#      - traefik.http.routers.elasticsearch-router.service=elasticsearch-service
+#      - traefik.http.services.elasticsearch-service.loadbalancer.server.port=9200
+    restart: always
+
+  keycloak:
+    image: 'jboss/keycloak:15.0.0'
+    container_name: keycloak
+    environment:
+      - "PROXY_ADDRESS_FORWARDING=true"
+      - 'DB_VENDOR=POSTGRES'
+      - 'DB_ADDR=postgres'
+      - 'DB_DATABASE=${POSTGRES_KEYCLOAK_DATABASE}'
+      - 'DB_USER=${POSTGRES_KEYCLOAK_USER}'
+      - 'DB_PASSWORD=${POSTGRES_KEYCLOAK_PASSWORD}'
+      - 'DB_SCHEMA=public'
+      - 'KEYCLOAK_USER=${KEYCLOAK_ADMIN_USERNAME}'
+      - 'KEYCLOAK_PASSWORD=${KEYCLOAK_ADMIN_PASSWORD}'
+      - 'KEYCLOAK_IMPORT=/tmp/keycloak-default-realm.json'
+      - 'JDBC_PARAMS="useSSL=false"' # No need for SSL between Postgres <-> Keycloak in a docker network
+#    ports:
+#      - '8445:${KEYCLOAK_PORT}'
+    depends_on:
+      - postgres
+    volumes:
+      - type: bind
+        source: $PWD/../common/keycloak-default-realm.json
+        target: /tmp/keycloak-default-realm.json
+      - type: volume
+        source: keycloak_logs
+        target: /opt/jboss/keycloak/standalone/log
+    logging:
+      driver: "json-file"
+      options:
+        max-size: "10m"
+        max-file: "10"
+        mode: non-blocking
+    labels:
+#      - traefik.http.routers.keycloak-router.tls=true
+      - traefik.http.routers.keycloak-router.entrypoints=http
+      - traefik.http.routers.keycloak-router.rule=PathPrefix(`/auth`)
+      - traefik.http.routers.keycloak-router.service=keycloak-service
+      - traefik.http.services.keycloak-service.loadbalancer.server.port=8080
+    restart: always
+
+  ui:
+    image: 'apiman/tomcat-manager:latest'
```

Comment:
Not sure about the latest.
 
I think we should suggest the latest release instead of latest because it could be the case that something is not working I guess.

Would do you think? Probably make a new variable like `APIMAN_VERSION`